### PR TITLE
added extra parameter to the SeQuant context which indentifies whether…

### PR DESCRIPTION
**Pull request by cmasteran1**
… or not spin free or spin basis is used for fermions. Context affects the symmetry of operators and the printing labels to_latex()

### CODE REVIEW

**Comment by evaleev** [comment]
@cmasteran1 just to make sure: this depends on #58 ?

**Comment by cmasteran1** [comment]
Yes those changes only make sense if the wick algorithm uses the tensor form

File: examples/uccf12/uccf12.cpp [hunk 1]
```diff
@@ -78,38 +79,89 @@ try_main() {
 
   // single commutator, needs symmetrization
   {
-    auto h = H();
+    auto h = H(false);
     auto r = R12(gg_space);
-    auto hr_comm = do_wick( ex<Constant>(2) * (h*r - r*h) );  // this assumes symmetrization includes 1/2
+    /*auto hr_comm = do_wick( ex<Constant>(2) * (h*r - r*h) );  // this assumes symmetrization includes 1/2
```
Comment (evaleev) [inline examples/uccf12/uccf12.cpp:84]: is this meant to be removed? Commented out code should be removed, unless used as documentation of something

File: examples/uccf12/uccf12.cpp [hunk 2]
```diff
@@ -47,7 +48,7 @@ try_main() {
     wick.spinfree(false)
         .full_contractions(false);
     auto result = wick.compute();
-    simplify(result);
+   simplify(result);
```
Comment (evaleev) [inline examples/uccf12/uccf12.cpp:51]: formatting seems off here and elsewhere ... do you not have auto-formatting ON in IDE?

**Review by evaleev — CHANGES_REQUESTED** [review]

**Pull request merged**